independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -2803,6 +2813,126 @@
         ]
       }
     },
+    "/crm/v3/objects/contacts/search": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "The default response.",
+            "schema": {}

```

Comment:
@ryanedshannon empty schema is not allowed. please define a proper schema.

Reply:
I defined the "ErrorModel" schema with the standard HubSpot structure and used it as a reference in the search operation. Let me know if that looks good.